File: source/AndroidClient/AtMiles/app/src/main/java/no/miles/atmiles/employee/SearchResultModel.java
package no.miles.atmiles.employee;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonTypeInfo;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;

@JsonIgnoreProperties(ignoreUnknown = true)
public class SearchResultModel {
    public int Total;
    public int Skipped;
    public Result[] Results;

    public static class Result{
        public String CompanyId;
        public String OfficeName;
        public String GlobalId;
        public String Name;
        public String DateOfBirth;
        public String JobTitle;
        public String PhoneNumber;
        public String Email;
        public String Thumb;
        public Address PrivateAddress;
    }

    public static class Address {
        public String Street;
        public String PostalCode;
        public String PostalName;
    }

}

File: source/Backend/Api/Models/Api/Search/SearchresultModel.cs
using System;
using System.Collections.Generic;

namespace no.miles.at.Backend.Api.Models.Api.Search
{
    public class SearchRequestModel
    {
        public string Query;
        public int Skip;
        public int Take;
    }

    public class SearchResultModel
    {
        public int Total;
        public int Skipped;
        public List<Result> Results;
    }

    public class Result
    {
        public string CompanyId;
        public string OfficeName;
        public string GlobalId;
        public string Name;
        public DateTime DateOfBirth;
        public string JobTitle;
        public string PhoneNumber;
        public string Email;
        public string Thumb;
        public Address PrivateAddress;

        public class Address
        {
            public string Street;
            public string PostalCode;
            public string PostalName;
        }
    }
}